web/client/components/map/openlayers/DrawSupport.jsx

```diff
@@ -475,11 +475,11 @@ export default class DrawSupport extends React.Component {
 
         this.removeSnapInteraction();
         if (mapLayerInstance) {
-            switch (mapLayerInstance.type) {
-            case "VECTOR":
+            switch (true) {
```

Comment:
This is completely wrong please change this
```suggestion
            switch (mapLayerInstance.type) {
```